util/utility.js

```diff
@@ -335,9 +327,9 @@ const getData = fromPromise(async (redis, url) => {
       timeout: url.timeout,
       retry: url.retries,
       hooks: {
-        afterResponse: [
-          async (response, retryWithMergedOptions) => {
-            if (isHypixelApi && !response.body.success) {
+        beforeRetry: [
+          async () => {
+            if (isHypixelApi) {
               const multi = redis.multi()
```

Comment:
When testing stuff I noticed that this multi implementation doesn't work at all, and has probably been broken since the `getData` rewrite.
![image](https://user-images.githubusercontent.com/14019974/81505530-850d2f80-92f8-11ea-85a5-399b026093fb.png)

Reply:
@Richienb do you want me to merge this PR now, you could address the issue in other one